### PR TITLE
Release fix

### PR DESCRIPTION
**Pull request by vguerci**

### CODE REVIEW

File: .github/workflows/release-cocoapods.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+name: Release on CocoaPods
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
```
Comment (TurpIF) [inline .github/workflows/release-cocoapods.yml:6]: Do you really want a workflow_dispatch for releasing ?

File: .github/workflows/release-cocoapods.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+          ${{ runner.os }}-gems-
+    - name: Bundle install
+      run: |
+        gem install bundler
+        bundle config path vendor/bundle
+        bundle install --jobs 4 --retry 3
+
+    - name: CocoaPods Release
+      run: |
+        bundle exec fastlane
```
Comment (TurpIF) [inline .github/workflows/release-cocoapods.yml:36]: bundle exec fastlane release_cocoapods ?

File: fastlane/Fastfile [hunk 3]
```diff
@@ -102,6 +102,9 @@ platform :ios do
 
   desc %(Archive then release version to GitHub and CocoaPods)
   lane :release do |options|
+    if ref = options[:ref]
+      options[:version] = ref.split("/").last
```
[the remark this answers is not in the document]
Comment (vguerci) [reply fastlane/Fastfile:106]: Done, also added filters on tag so release isn't triggered on non semver branches

File: fastlane/Fastfile [hunk 4]
```diff
@@ -105,7 +105,6 @@ platform :ios do
     if ref = options[:ref]
       options[:version] = ref.split("/").last
     end
-    archive
```
[the remark this answers is not in the document]
Comment (vguerci) [reply fastlane/Fastfile:105]: This works locally but for some reason not on GitHub, I'd like to fix this someday to be GitHub agnostic as possible

File: .github/workflows/release-cocoapods.yml (hunk 1 above)
Comment (vguerci) [reply .github/workflows/release-cocoapods.yml]: This is handy to test workflow, as publishing a release is not the easiest thing to do. Anyway, removed.

File: .github/workflows/release-cocoapods.yml (hunk 2 above)
Comment (vguerci) [reply .github/workflows/release-cocoapods.yml]: Fixed

**Comment by vguerci** [comment]
Yup but notice 40m before another one only to rebase to prevent noise in the later ;)

**Pull request merged**